src/modules/dataTypes.js

```diff
@@ -6,6 +6,8 @@ export const DIMENSION_TYPE_DATA_ELEMENT = 'DATA_ELEMENT'
 export const DIMENSION_TYPE_DATA_SET = 'DATA_SET'
 export const DIMENSION_TYPE_EVENT_DATA_ITEM = 'EVENT_DATA_ITEM'
 export const DIMENSION_TYPE_PROGRAM_INDICATOR = 'PROGRAM_INDICATOR'
+export const DIMENSION_TYPE_EXPRESSION_DIMENSION_ITEM =
```

Comment:
Im a bit surprised that GH doesn't pick up that this change already exists in my branch (https://github.com/dhis2/analytics/pull/1370/files?show-viewed-files=true&file-filters%5B%5D=#diff-64e11558ada84b560c86145d9ac2a5eb6c83a7f24fc40ef088b7f40aa2cbec48R21) 🤔

Reply:
Moved it to the same line, but still not picked up.